src/verifiable-credential.js

```diff
@@ -9,6 +9,7 @@ import {
   ensureURI,
```

Comment:
What about uniqueness of subject?

Reply:
Good catch! The issue wasn't about it but I can add it here.

Reply:
If by ticket you mean clubhouse then the title of the ticket mentions subject as well.